.webpack/webpack.config.prod.js

```diff
@@ -4,7 +4,10 @@ const { filePath } = helpers;
 
 module.exports = presets.production( {
     name: 'before-you-go',
-    externals,
+    externals: {
```

Comment:
Hah, the inconsistency is mine, I see... these files didnt' start out as tabbed indent at all. Wacky.